controllers/expression.js

```diff
@@ -0,0 +1,63 @@
+import expressionService from '../service/expression';
+
+class ExpressionController {
+  constructor(expressionService) {
+    this._expressionService = expressionService;
+
+    this.addExpression = this.addExpression.bind(this);
```

Comment:
Imagine if you have 10 functions in this Controller. Are you going to bind them all here? I left you a comment with an shorter way of doing this